Atlassian.Jira/Remote/ProjectService.cs

```diff
@@ -28,5 +29,23 @@ public async Task<Project> GetProjectAsync(string projectKey, CancellationToken
             var remoteProject = await _jira.RestClient.ExecuteRequestAsync<RemoteProject>(Method.GET, resource, null, token).ConfigureAwait(false);
             return new Project(_jira, remoteProject);
         }
+
+        public async Task<(IEnumerable<IssueType>, IEnumerable<ProjectComponent>)>

```

Comment:
Naming the return type can improve readability and prevent the use of `ItemN` when not deconstructing. something like this:

```suggestion
        public async Task<(IEnumerable<IssueType> issueTypes, IEnumerable<ProjectComponent> project)>

```